pkg/command/service/generate.go

```diff
@@ -87,9 +86,6 @@ kperf service generate —n 500 —-interval 20 —-batch 20 --min-scale 0 --max
 				nsNameList = append(nsNameList, generateArgs.namespace)
 			}
 
-			if len(nsNameList) == 0 {
-				return fmt.Errorf("no namespace found with prefix %s", generateArgs.namespacePrefix)
-			}
```

Comment:
@gyohuangxin why this part is removed? I think it is a double check before iteration

Reply:
I think this part will never executed, if `generateArgs.namespacePrefix == "" && generateArgs.namespace == ""`, len(nsNameList) = 1. If `generateArgs.namespace != ""`, len(nsNameList) = 1. And if `generateArgs.namespacePrefix != ""`, len(nsNameList) = 1.

But keeping it for a double check is also fine.